packages/hooks/package-lock.json

```diff
@@ -119,12 +119,6 @@
         "chalk": "^4.0.0"
       }
     },
-    "@jobber/formatters": {
-      "version": "0.2.0",
-      "resolved": "https://registry.npmjs.org/@jobber/formatters/-/formatters-0.2.0.tgz",
-      "integrity": "sha512-RraFVvzBm+fRMTMtTYLWgpUSWDitKwTGocTsZsHttsu1Rvssg+E4k/XL/AUkB4L6j3kQp5mOf/F2fQ2/bay8sA==",
-      "dev": true
-    },
```

Comment:
We don't need this one anymore?

Reply:
it's autogenerated so if the lerna says no, then it's a no haha